.circleci/config.yml

```diff
@@ -116,11 +116,11 @@ install_dep_pytorch_nightly: &install_dep_pytorch_nightly
         # check if we have restored venv cache (/home/circleci/venv) correctly, if so, just skip
         if [ -f /home/circleci/venv/check_version.py ]; then python /home/circleci/venv/check_version.py torch eq 1.10 && exit 0; fi
         # start installing
-        pip install --progress-bar off --pre torch==1.10.0.dev20210901+cu111 torchvision==0.11.0.dev20210901+cu111 -f https://download.pytorch.org/whl/nightly/cu111/torch_nightly.html
+        pip install --progress-bar off --pre torch==1.11.0.dev20211018+cu111 torchvision==0.12.0.dev20211018+cu111 -f https://download.pytorch.org/whl/nightly/cu111/torch_nightly.html
```

Comment:
cc @min-xu-ai @anj-s , checking that this is ok

Reply:
hmm, looks like the change should be unrelated to this PR, something is not right

Reply:
I did an interactive rebase of a few older commits than my commit and then pushed the branch in. I think these changes are showing up in the PR now as a side-effect. I will try to remove those commits from the PR to make it cleaner to review